homeassistant/components/persistent_notification/__init__.py

```diff
@@ -134,6 +135,7 @@ def create_service(call):
             message = message.template
 
         attr[ATTR_MESSAGE] = message
+        attr[ATTR_FRIENDLY_NAME] = humanize_entity_name(notification_id)
```

Comment:
Should it be the title?

Reply:
That would also be an option...probably the better one.